lab-scott/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const getFiles = require(__dirname + '/lib/getFiles');
+const fs = require('fs');
+
+const filePaths = fs.readdir('./data/', function(err, files) {
```

Comment:
@slugbyte I was reading a bit on this today, and it sounds like `fs.readdir()` is reliably sorting the files alphabetically on OSX, but not other OS's. Any thoughts? Better to just be explicit about the file paths?